extras/rya.geoindexing/src/test/java/org/apache/rya/indexing/geotemporal/model/EventQueryNodeTest.java

```diff
@@ -113,7 +113,7 @@ public void constructor_variablePredicate() throws Exception {
 
     @Test
     public void evaluate_constantSubject() throws Exception {
```

Comment:
Maybe not your doing, but why does every test grab the mongo client?  Why not just use MongoITBase?